### DISCUSSION TITLE
生成vs工程如何控制生成的平台

### DISCUSSION THREAD

**Discussion by RI-ss** [xmake-io/xmake#3394] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/71321125/219925819-d8af4b4e-04f5-42df-ac0a-6058c6e54bcb.png)
我不想生成arm平台的，请问如何实现

**Comment by waruqi** [comment, marked as answer]
你的版本不是最新的，最新版本应该默认不生成 arm 了。。另外可以指定 arch

xmake project -k vsxmake -a x64 -m release
